packages/cli-main/src/cli/services/commands/version.test.ts

```diff
@@ -40,7 +39,9 @@ describe('check CLI version', () => {
       const outputReminder = vi.mocked(output.getOutputUpdateCLIReminder).mockReturnValue('CLI reminder')
 
       // When
-      await Version.run()
+      await expect(async () => {
+        await versionService({currentVersion})
```

Comment:
Is the inner await needed? 🤔

Reply:
Alternatively we can return the promise:
```
return versionService({currentVersion})
```